newsletter/index.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <title>Origo newsletter</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+  <ul>
+    <h1>Origo newsletter downloads</h1>
+    <li>
+      <a href="newsletters/20190206.pdf" download>2019-02-16</a>
+    </li>
+  </ul>
+</body>
+
+</html>
```

Comment:
Missing newline

Reply:
@markus659 Im not following you on this one. Could you specify?

Reply:
Usually files will end with a newline, if you look at https://github.com/origo-map/origo-map.github.io/pull/1/files you will see GitHub reporting the missing newline (also the css-file). In most code editors you can set it so that when you save the file it automatically adds a newline at the end of the file.

Reply:
@markus659 Ahh, thank you for the explanation Markus